pkg/parser/parser.go

```diff
@@ -543,6 +547,14 @@ func (p *parser[T]) ColIntGetter(colName string) (func(any) int64, error) {
 		return nil, fmt.Errorf("column %s not found", colName)
 	}
 
+	// TODO: Handle all int types?
+	if col.HasCustomExtractor() && col.RawType().Kind() == reflect.Int64 {
+		f := columns.GetFieldFuncExt[int64, T](col, true)
+		return func(a any) int64 {
+			return f(a.(*T))
+		}, nil
+	}
```

Comment:
JFYI: I'm working on this.

Reply:
Just adding another idea, not saying we should go for it:

4. Return a pointer so that it also can handle `nil` - in which case the TextFormatter would return an empty string. But that would require supporting pointer data types for everything, which complicates things a bit.

I think implementing something like `omitempty` could quickly limit us when we actually _do_ want to handle/print a zero value. Thinking more about it...

Reply:
After sleeping on it, I think implementing support for `omitempty` should be the right approach. This is the perfect example for that: we have a field that we don't want to print when its value is zero. This is also very useful for containerized gadgets as in that case users can't implement custom logic to print the value. 

I agree that there could be cases that aren't supported by it, but I don't think we should block trying to find a very generic solution when there is one that works fine for this case.